test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/JsonPatchTests.cs

```diff
@@ -301,5 +301,22 @@ public async Task GivenAnExistingResourceWithMismatchingData_WhenPatchingWithPat
 
             Assert.Equal(HttpStatusCode.BadRequest, exception.Response.StatusCode);
         }
+
+        [Fact]
+        [Trait(Traits.Priority, Priority.One)]
+        public async Task GivenAPatchDocumentWithDateTime_WhenDateTimeHasOffset_ThenOffsetShouldBePreserved()
+        {
+            var poco = Samples.GetJsonSample("PatientWithMinimalData").ToPoco<Patient>();
+            FhirResponse<Patient> response = await _client.CreateAsync(poco);
+
+            string dateTimeOffsetString = "2022-05-02T14:00:00+02:00";
+            string patchDocument =
+                $@"[{{""op"":""add"",""path"":""/deceasedDateTime"",""value"":""{dateTimeOffsetString}""}}]";
```

Comment:
Our coding standard for the repo is to not use string interpolation. Can you match the formatting to [this line](https://github.com/microsoft/fhir-server/blob/bb843d0f00d00dfdeb55b3c7d20629ec6f112bbc/test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/JsonPatchTests.cs#L151)?

Reply:
Done!